css/css/master.scss

```diff
@@ -0,0 +1,194 @@
+        background: color("brand-orange");
+        cursor: pointer;
+        transition: background time("hover");
+        &:hover{
+            background: darken(color("brand-orange"),6%);
+            svg{
+                transform:rotate(360deg);
+            }
+        }
+        svg{
+            position:absolute;
+            top:0.2rem;
+            left:0.2rem;
+            width: 2.5rem;
+            height: 2.5rem;
+            fill: white;
+            transition: transform time("hover");
+        }
+    }
+    .heart{
+        position:absolute;
+        top:1.6rem;
+        right:1.6rem;
+        width: 3rem;
+        height: 3rem;
+        background: rgba(color("background"),0.85);
+        opacity: 0;
+        border-radius: 50%;
+        transition: opacity time("hover"), background time("hover");
+        cursor: pointer;
+        z-index: layer("topContent");
+        &:before{
+            content:"";
+            position:absolute;
+            top:0;
+            left:0;
+            width:100%;
+            height:100%;
+            border-radius: 50%;
+            border:solid 3px color("brand-red");
+            transform: scale(0.95);
+            opacity:0;
+            z-index:-1;
+        }
+        &:hover{
+            background: color("background");
+        }
+        &.favorite{
+            background: color("brand-red");
+            &:before{
+                animation: heartCircle 0.5s;
+            }
+            &:first-of-type{
+                opacity:0;
+            }
+            svg:nth-of-type(2){
+                opacity:1;
+            }
+        }
+        svg{
+            position:absolute;
+            top:0.9rem;
+            left: 50%;
+            width: 1.25rem;
+            height: 1.25rem;
+            transform: translateX(-50%);
+            transition: opacity time("hover");
+            &:first-of-type{
+                fill:color("brand-gray-dark");
+            }
+            &:nth-of-type(2){
+                opacity:0;
+                fill:white;
+            }
+        }
+    }
+    figure{
+        width:100%;
+        img{
+            width:100%;
+        }
+        figcaption{
+            display:inline-block;
+            color: color("brand-green");
+            font-size: fontSize("title");
+            font-weight: 900;
+            border-bottom: solid 2px;
+            padding:0.625rem 0 2px;
+            margin-bottom:1rem;
+        }
+        span.time{
+            display:inline-block;
+            position:relative;
+            float:right;
+            margin-top: 0.625rem;
+            font-size: fontSize("body");
+            font-weight: 500;
+            color: color("brand-gray-xlight");
+            svg{
+                position:relative;
+                top: 0.125rem;
+                width:1rem;
+                height:1rem;
+                fill: color("brand-gray-xlight");
+            }
+        }
+    }
+    aside{
```

Comment:
In general, it's not the best idea to to target elements directly. In the short term it may seem like a good idea, but it almost always leads to specificity issues and many time limits the ability to reuse components in different contexts.